product-matrix.json

```diff
@@ -22,14 +12,14 @@
       "untilBuild": "183.*"
     },
     {
-      "comments": "IntelliJ 2019.1, AS 3.5 canary",
+      "comments": "IntelliJ 2019.1.1, AS 3.5 canary",
       "name": "Android Studio 3.5",
       "version": "3.5",
-      "ijVersion": "2019.1",
+      "ijVersion": "2019.1.1",
       "ideaProduct": "android-studio",
-      "ideaVersion": "191.5416148",
-      "dartPluginVersion": "191.6183.87",
-      "sinceBuild": "191.5416148",
+      "ideaVersion": "191.5471097",
```

Comment:
Is rev'ing this intentional (and, will it leave behind users)? We should just verify what version people are using internally (2019.1, or 2019.1.1).

Reply:
That's the build artifact, and it is based on 2019.1.